src/main/java/net/sf/jabref/event/ChangeFieldEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.sf.jabref.event;
+
+
+public class ChangeFieldEvent {
```

Comment:
I vote for renaming to `FieldAddedChangedRemovedEvent`, because this event is also fired when a field is removed or added.